core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpReceiverFuzzTest.java

```diff
@@ -0,0 +1,360 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cutlass.line.tcp;
+
+import io.questdb.cairo.TableReader;
+import io.questdb.cairo.TableReaderMetadata;
+import io.questdb.cairo.TableWriter;
+import io.questdb.cairo.pool.PoolListener;
+import io.questdb.cairo.security.AllowAllCairoSecurityContext;
+import io.questdb.cutlass.line.tcp.load.LineData;
+import io.questdb.cutlass.line.tcp.load.TableData;
+import io.questdb.log.Log;
+import io.questdb.log.LogFactory;
+import io.questdb.mp.SOCountDownLatch;
+import io.questdb.mp.WorkerPoolConfiguration;
+import io.questdb.std.CharSequenceObjHashMap;
+import io.questdb.std.Os;
+import io.questdb.std.Rnd;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static io.questdb.cairo.ColumnType.DOUBLE;
+import static io.questdb.cairo.ColumnType.STRING;
+
+public class LineTcpReceiverFuzzTest extends AbstractLineTcpReceiverTest {
+    private static final Log LOG = LogFactory.getLog(LineTcpReceiverFuzzTest.class);
+
+    private static final int MAX_NUM_OF_SKIPPED_COLS = 2;
+    private static final int NEW_COLUMN_RANDOMIZE_FACTOR = 2;
+
+    private final Rnd random = new Rnd();
```

Comment:
nit: IMO we should initialize the random generator with the system clock timestamp here to have non-deterministic randomization. Otherwise, this test will be always replaying the same payload which is orthogonal to fuzz testing.

Reply:
good point, let me change it.
thank you